### DISCUSSION TITLE
Migration path from AWS PDK and typeSafeApi

### DISCUSSION THREAD

**Discussion by Tameem-623** [aws/aws-pdk#885] [Q&A]
I'm working on migrating our application away from AWS PDK and typeSafeApi. Looking for guidance on the best approach to handle this transition smoothly.

### Current Setup

- Using AWS PDK for infrastructure management
- Using TypeSafeApiProject for APIs
- Using CloudscapeReactTsWebsiteProject

### Migration Goals

- Move away from AWS PDK to a more maintainable solution (AWS CDK)

**Comment by cogwirrel** [comment]
Hello - you may wish to consider the Nx Plugin for AWS. We have a migration guide [here](https://awslabs.github.io/nx-plugin-for-aws/en/blog/migrating-from-aws-pdk/) which may help.
